data/org.freedesktop.UDisks2.xml

```diff
@@ -2026,6 +2026,15 @@
     -->
     <property name="ChildConfiguration" type="a(sa{sv})" access="read"/>
 
+    <!-- HintEncryptionType:
+         If not blank, the type of the encryption used to encrypt this device.
+
+         This is set during successful unlocking of an encrypted device, whose encryption
+         type can only be determined by decrypting the device (currently only used
+         for TCRYPT devices).
```

Comment:
Why should this not be used for ``LUKS`` too? There's no reason to enforce checking multiple properties to get an encryption type.

Reply:
The reason I added this property is that `TCRYPT` volumes cannot be reliably detected as such. To make it less confusing to users, I set the `IdType` property to `crypto_unknown` for volumes which might be `TCRYPT` encrypted. But after a volume was successfully unlocked as `TCRYPT`, we know for sure that it's `TCRYPT`, so we set the `HintEncryptionType` property. And then, when the interface is updated, the `IdType` is set according to `HintEncryptionType`, i.e. to `crypto_TCRYPT` (see https://github.com/storaged-project/udisks/pull/495/files#diff-fd982868b9106b4c67ff22c3d8bd7179R138).

Doing the same for `LUKS` doesn't make sense, because the `IdType` is always set to `crypto_LUKS`.

Reply:
Yes, you don't need it for LUKS, but I think it can be really confusing to have a property `HintEncryptionType` that says `NULL` for LUKS devices (or actually all non-truecrypt devices). I think that having this information twice is better.

Reply:
Ok, done in b913bf796b0c2bc46241f98298714a528d56bb38.

Reply:
Looks like the documentation here didn't get fixed up in the above commit.